routers/web/repo/setting/webhook.go

```diff
@@ -149,7 +149,7 @@ func WebhooksNew(ctx *context.Context) {
 			"Username": "Gitea",
 		}
 	}
-	ctx.Data["BaseLink"] = orCtx.LinkNew
+	ctx.Data["BaseLinkNew"] = orCtx.LinkNew
```

Comment:
Why we need a new name?

Reply:
Because the `BaseLinkNew` variable read uniformly in the type selection module (previously in `templates/repo/settings/webhook/base_list.tmpl`, now pulled out to become the common module `templates/webhook/selector.tmpl`)